test/checkkeys.c

```diff
@@ -221,6 +221,12 @@ main(int argc, char *argv[])
         quit(2);
     }
 
+    /* On wayland, no window will actually show until something has
+       actually been displayed.
+    */
+    SDL_Renderer *renderer = SDL_CreateRenderer(window, -1, 0);
+    SDL_RenderPresent(renderer);
```

Comment:
This will hit `-Werror=declaration-after-statement`

Reply:
Whoops. I've pushed a new version that fixes it.